app/controllers/divisions_controller.rb

```diff
@@ -219,4 +233,11 @@ def get_member
           .where(constituency: electorate)
           .order(entered_house: :desc).first
   end
+
+  def get_division(house, date, number)
```

Comment:
If you're not fussed neither am I. We can always refactor later if the smell gets stronger :smile: